### PR TITLE
Isar's Cross waterfall animations

### PR DESCRIPTION
**Pull request by inferno8**
This addressees #7348. The PR includes 40 waterfall animation frames for Isar's Cross and the updated scenario file. It also adds a new "Art Placeholder" (^Xxxx) terrain, which is used to place the animation.

The looped animation was rendered at 24fps (.webp RGBA format.).

isar waterfall

### CODE REVIEW

**Review by knyghtmare — APPROVED** [review]
Nice. No arguments from me. This goes in.

**Comment by ForestDragon-wesnoth** [comment]
Nice animation, I like it

**Comment by Pentarctagon** [comment]
Looks great!

I do wonder if a different terrain code would be better though, or at least one that has a pattern that can be reused (ie: Xxxa so Xxxb, Xxxc, etc can be added if needed for other things).

File: data/multiplayer/scenarios/4p_Isars_Cross.cfg [hunk 1]
```diff
@@ -92,9 +96,103 @@
...
...ific/isar-waterfall/18.webp:42,map-specific/isar-waterfall/19.webp:42,map-specific/isar-waterfall/20.webp:42,map-specific/isar-waterfall/21.webp:42,map-specific/isar-waterfall/22.webp:42,map-specific/isar-waterfall/23.webp:42,map-specific/isar-waterfall/24.webp:42,map-specific/isar-waterfall/25.webp:42,map-specific/isar-waterfall/26.webp:42,map-specific/isar-waterfall/27.webp:42,map-specific/isar-waterfall/28.webp:42,map-specific/isar-waterfall/29.webp:42,map-specific/isar-waterfall/30.webp:42,map-specific/isar-waterfall/31.webp:42,map-specific/isar-waterfall/32.webp:42,map-specific/isar-waterfall/33.webp:42,map-specific/isar-waterfall/34.webp:42,map-specific/isar-waterfall/35.webp:42,map-specific/isar-waterfall/36.webp:42,map-specific/isar-waterfall/37.webp:42,map-specific/isar-waterfall/38.webp:42,map-specific/isar-waterfall/39.webp:42,map-specific/isar-waterfall/40.webp:42,map-specific/isar-waterfall/41.webp:42,map-specific/isar-waterfall/42.webp:42,map-specific/isar-waterfall/43.webp:42,map-specific/isar-waterfall/44.webp:42,map-specific/isar-waterfall/45.webp:42,map-specific/isar-waterfall/46.webp:42,map-specific/isar-waterfall/47.webp:42,map-specific/isar-waterfall/48.webp:42
```
Comment (CelticMinstrel) [inline data/multiplayer/scenarios/4p_Isars_Cross.cfg:107]: Can't this use progressive notation?

`map-specific/isar-waterfall/[01~48].webp:42`

Comment (inferno8) [reply data/multiplayer/scenarios/4p_Isars_Cross.cfg]: That's a good point.

File: data/multiplayer/scenarios/4p_Isars_Cross.cfg [hunk 2]
```diff
@@ -104,7 +104,7 @@
...
+            name=map-specific/isar-waterfall/[01~40].webp:42
```
Comment (CelticMinstrel) [inline data/multiplayer/scenarios/4p_Isars_Cross.cfg:107]: Wasn't it 48 not 40?

Comment (inferno8) [reply data/multiplayer/scenarios/4p_Isars_Cross.cfg:107]: 48 was my mistake - there are 40 frames total.

**Review by doofus-01 — APPROVED** [review]
Gif looks good!  If something needs cleanup or optimization, there's time to fix it before 1.18

**Pull request merged**